web/modules/custom/portland/modules/portland_location_picker/js/portland_location_picker.js

```diff
@@ -791,31 +945,207 @@
+              }
+              if (results.park) {
+                $('input[name=' + elementId + '\\[location_type_park\\]]').val("1");
+                location_type += "park,";
+                var park_name = results.feature_detail.park[0].name;
+                place_name += park_name + ', ';
+                type_count += 1;
+              }
+              if (results.waterbody) {
+                $('input[name=' + elementId + '\\[location_type_waterbody\\]]').val("1");
+                location_type += "waterbody,";
+                var waterbody_id = results.feature_detail.waterbody[0].objectid;
+                //place_name += waterbody_id + ', ';
+                internal_details += "Object ID: " + waterbody_id + ', ';
+                type_count += 1;
+              }
+              if (results.trail) {
+                $('input[name=' + elementId + '\\[location_type_trail\\]]').val("1");
+                location_type += "trail,";
+                var trail_name = results.feature_detail.trail[0].local_name;
+                place_name += trail_name + ', ';
+                type_count += 1;
+              }
+              if (results.stream) {
+                $('input[name=' + elementId + '\\[location_type_stream\\]]').val("1");
+                location_type += "stream,";
+                var stream_name = results.feature_detail.stream[0].name;
+                place_name += stream_name + ', ';
+                type_count += 1;
+              }
+              if (results.street) {
+                $('input[name=' + elementId + '\\[location_type_street\\]]').val("1");
+                location_type += "street,";
+                var street_name = results.feature_detail.street[0].full_name;
+                place_name += street_name + ', ';
+                type_count += 1;
+              }
+              if (results.row) {
+                $('input[name=' + elementId + '\\[location_type_row\\]]').val("1");
+                // $('input[name=' + elementId + '\\[location_type\\]]').val($('input[name=' + elementId + '\\[location_type\\]]').val() + 'row,');
+                location_type += "row,";
+                var tl_type = results.feature_detail.row[0].tl_type;
+                internal_details += "TL Type: " + tl_type + ', ';
+                type_count += 1;
+              }
+
+              place_name = removeTrailingComma(place_name);
+              location_type = removeTrailingComma(location_type);
+              internal_details = removeTrailingComma(internal_details);
+
+              //$('input[name=' + elementId + '\\[place_name\\]]').val(place_name);
+              $('input[name=' + elementId + '\\[location_type_hidden\\]]').val(location_type);
+              $('input[name=' + elementId + '\\[location_attributes\\]]').val(internal_details);
+
+              // set location radio button
+              // report_location[location_type]
+              var radioInputs = $('input[name="report_location[location_type]"]');
```

Comment:
This should probably use `elementId`.

Reply:
Fixed. Good catch!

Reply:
This is strange. I've updaed and pushed the file to fix this issue, but the PR is still showing the old version. Though I can see the commit with the change in the log.

Reply:
@odensc can you see this as fixed on line 1068?

Reply:
@jookyg Yup, it shows up for me. Super weird! I guess it's fine if I can see it 😁